ros/src/msgs/autoware_config_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(autoware_config_msgs)
+
+find_package(catkin REQUIRED COMPONENTS
+  message_generation
+  std_msgs
+)
+
+## Generate messages in the 'msg' folder
+add_message_files(
+  DIRECTORY msg
+  FILES
+
+  ## Runtime Manager ##
+  ConfigApproximateNdtMapping.msg
+  ConfigCarDpm.msg
+  ConfigCarFusion.msg
+  ConfigCarKf.msg
+  ConfigCompareMapFilter
```

Comment:
Thanks! Adding .msg fixed the Travis error.

Reply:
Fixed. Thanks!